cli-commands/commands.js

```diff
@@ -477,7 +491,144 @@ const commands = [
 				logger.removeOutputStorage();
 			}
 		}
-	}
+	},
+	{
+		command: 'circuit-compile',
+		description: 'Compile a circuit file located in zero-knowledge-proof/circuits',
+		argumentsProcessor: (yargs) => {
+
+		},
+		commandProcessor: async (argv) => {
+			try {
+				await circuitCompile.run();
+			} catch (err) {
+				console.error(err);
+			} finally {
+				logger.removeOutputStorage();
+			}
+		}
+	},
+	{
+		command: 'trusted-setup',
+		description: 'Establish a trusted setup based on circuit and generates prooving key and verification key',
+		argumentsProcessor: (yargs) => {
+
+		},
+		commandProcessor: async (argv) => {
+			try {
+				await trustedSetup.run();
+			} catch (err) {
+				console.error(err);
+			} finally {
+				logger.removeOutputStorage();
+			}
+		}
+	},
+	{
+		command: 'proof [signal] [circuit] [proving_key]',
+		description: 'Establish a trusted setup based on circuit and generates prooving key and verification key',
+		argumentsProcessor: (yargs) => {
+			yargs.positional('signal', {
+				describe: 'Specifies the file with signals to be used for generating a proof. Defaults to input.json',
+				type: 'string',
+				default: 'input.json'
+			});
+			yargs.positional('circuit', {
+				describe: 'Specifies the compiled circuit for checking of matched signals. Defaults to: circuit.json',
+				type: 'string',
+				default: 'circuit.json'
+			});
+			yargs.positional('proving_key', {
+				describe: 'Specifies the prooving key to be used for generating a proof. Defaults to: circuit_proving_key.json',
+				type: 'string',
+				default: 'circuit_proving_key.json'
+			});
+		},
+		commandProcessor: async (argv) => {
+			try {
+				await proof.run(argv.signal, argv.circuit, argv.proving_key);
+			} catch (err) {
+				console.error(err);
+			} finally {
+				logger.removeOutputStorage();
+			}
+		}
+	},
+	{
+		command: 'verify [public_signals] [proof] [verifier_key]',
```

Comment:
I think all commands for zk should either have a zk- preffix or all of them should be part of one command `etherlime zk` and add parameters like `verify` or `proof` resulting in `etherlime zk proof` and `etherlime zk verify`.

Reason being is that if we want to build a feature for verification of smart contracts in etherscan, we will now have no way to name it as `etherlime verify` would be taken